ipmininet/router/config/base.py

```diff
@@ -35,7 +39,7 @@ def __init__(self, node, daemons=(), sysctl=None,
                        interfaces."""
         self._node = node  # The node for which we will build the configuration
         self._daemons = {}  # Active daemons
-        map(self.register_daemon, daemons)
+        list(map(self.register_daemon, daemons))
```

Comment:
The map call there was a (pointless) way to make this a one-liner instead of a simple for loop.
You definitely do not need the list expansion (but still need to run the loop).

I'd rewrite all such case by their for loop equivalent ...

```python
for d in daemons:
    self.register_daemon(d)
```